cloudron/template.md

```diff
@@ -0,0 +1,49 @@
+## App Information
+
+### App Name
+
+Cloudron
+
+### App description
+
+Cloudron is a turnkey solution for running apps like WordPress, Rocket.Chat, NextCloud, GitLab, OpenVPN & many more. Cloudron performs end-to-end deployment of apps including provisioning databases, automated DNS setup, certificate management, centralized user management, periodic backups. Apps on Cloudron also receive automatic updates saving you the hassle of tracking upstream releases and keeping the installation secure.
+
+### Version Number
+
+5.6.0
+
+### Support URL
+
+* [Forum](https://forum.cloudron.io)
+* [Documentation](https://docs.cloudron.io)
+* [Email](mailto:support@cloudron.io)
+
+### Operating System
+
+Ubuntu 18.04 LTS

```

Comment:
Any reason why this can't be 20.02 LTS ? where can one follow this issue ?

Reply:
We are working on it for Cloudron 6. It will be a major upgrade for our users . You can follow it here - https://git.cloudron.io/cloudron/box/-/issues/684 or https://forum.cloudron.io/topic/3013/support-ubuntu-20-04-lts . Last we checked there was a bug in the python3+collectd integration in Focal.